frontend/server/src/main/java/org/pytorch/serve/wlm/AsyncWorkerThread.java

```diff
@@ -75,6 +77,14 @@ public void run() {
                 try {
                     backendChannel.get(0).writeAndFlush(req).sync();
                     logger.debug("Successfully flushed req");
+
+                    if (loadingFinished == false) {
```

Comment:
Is this specific to async workers only?

Reply:
Yes, sync worker is waiting for model LOAD command to complete but async worker does not.